### PR TITLE
First commit Dani

### PR DESCRIPTION
**Pull request by daniscript**
My first fork

### CODE REVIEW

File: staff/daniel-ferrandez-latre/precourse/tema1_calculadora/index.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    }
+}  
+
+function operationPicker() {
+    var inputPickOperation = arguments[0];
+    if(RegExIsSquareRoot.test(inputPickOperation)) {
+        console.log('+ Processing square root...');
+        return false;
+    } else if (RegExIsOperation.test(inputPickOperation)) {
+        console.log('+ Porcessing operation...');
```
Comment (paulusrex) [inline staff/daniel-ferrandez-latre/precourse/tema1_calculadora/index.js]: Bad spelling "processing"

File: staff/daniel-ferrandez-latre/precourse/calculadora2/main.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        console.log(calculator.arrayOpertaion.join(""));
+    }
+
+    document.addEventListener("click", function() {
+        document.getElementById("showOperation").innerHTML = calculator.operationResult;
+    });
+
+    window.onclick = e => {
+        if(e.target.tagName == "DIV"){
+            document.getElementById("showOperation").innerHTML = calculator.arrayOpertaion.join("");
```
Comment (edfrias) [inline staff/daniel-ferrandez-latre/precourse/calculadora2/main.js]: @DaniLatreCodes arrayOpertaion = OpertAIOn! I think you mean arrayOperation, right? :P

File: staff/daniel-ferrandez-latre/precourse/tema1_calculadora/index.js [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    var isMultDiv = false;
+    if(arrayOperationMultDivResult.includes('*', '/')){
+        for(let i = 0; i < arrayOperationMultDivResult.length; i++){
+            isMultDiv = RegExIsMultDiv.test(arrayOperationMultDivResult[i]);
+            if(isMultDiv){
+                Operation.num1 = arrayOperationMultDivResult[i - 1];
+                Operation.num2 = arrayOperationMultDivResult[i + 1];
+                Operation.operator = arrayOperationMultDivResult[i];
+                arrayOperationMultDivResult.splice(i - 1, 3, Operation.doSingleOperation());
+                i = i - 2;
```
Comment (edfrias) [inline staff/daniel-ferrandez-latre/precourse/tema1_calculadora/index.js]: @DaniLatreCodes I think you can do this here: i-=2;

**Review by bilalelbedoui93 — COMMENTED** [review]
well done

File: staff/daniel-ferrandez-latre/precourse/calculadora2/main.js (hunk 2 above)
Comment (daniscript) [reply staff/daniel-ferrandez-latre/precourse/calculadora2/main.js]: Rigth, Im on it!!!

File: staff/daniel-ferrandez-latre/precourse/tema1_calculadora/index.js (hunk 3 above)
Comment (daniscript) [reply staff/daniel-ferrandez-latre/precourse/tema1_calculadora/index.js]: Yea more elegant approach

File: staff/daniel-ferrandez-latre/precourse/tema1_calculadora/index.js (hunk 1 above)
Comment (daniscript) [reply staff/daniel-ferrandez-latre/precourse/tema1_calculadora/index.js]: Sovel!! ;)

**Pull request merged**